### PR TITLE
CAS can no longer be launched before shutters open, or before Tadpole launches

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Per title.

## Why It's Good For The Game
I am getting so tired of the classic PO griefing the round for whatever reason.

## Changelog
:cl: Lewdcifer
fix: CAS is no longer usable before shutters open, or before Tadpole launches.
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
Speedmerge please

**Comment by maikilangiolo** [comment]
You forget POs and CMOs that grief by launching tadpole before shutters, too

**Comment by lKiseki** [comment]
> You forget POs and CMOs that grief by launching tadpole before shutters, too

I am told Tadpole already has a time lock.

**Comment by novaepee** [comment]
after 3 years of dealing with this
finally, the pr is here

File: code/modules/shuttle/cas_plane.dm [hunk 1]
```diff
@@ -42,6 +45,13 @@
 	if(occupant)
 		to_chat(occupant, span_notice("CAS laser detected. Target: [AREACOORD_NO_Z(incoming_laser)]"))
 
+/obj/structure/caspart/caschair/proc/cas_usable(datum/source)
+	SIGNAL_HANDLER
+	cas_usable++
```
Comment (Lumipharon) [inline code/modules/shuttle/cas_plane.dm:50]: ? Just make it true.
Also are you sure all these signals are needed? The one labeled crash for example, is that not actually related to the crash gamemode?

File: code/modules/shuttle/cas_plane.dm [hunk 2]
```diff
@@ -21,11 +21,14 @@
 	var/mob/living/carbon/human/occupant
 	///Animated cockpit /image overlay, 96x96
 	var/image/cockpit
+	/// Whether CAS is usable or not.
+	var/cas_usable = 0
```
Comment (Lumipharon) [inline code/modules/shuttle/cas_plane.dm:25]: This doesn't need to be defined, since that makes it null anyway. Otherwise it can be FALSE

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/cas_plane.dm (hunk 2 above)
Comment (lKiseki) [reply code/modules/shuttle/cas_plane.dm:25]: Gotcha.

File: code/modules/shuttle/cas_plane.dm (hunk 1 above)
Comment (lKiseki) [reply code/modules/shuttle/cas_plane.dm:50]: Made it true.
Also, you'd be right on the crash signal. Every other signal is necessary, however.
